tests/unit/splinterdb_quick_test.c

```diff
@@ -67,6 +67,10 @@ check_current_tuple(splinterdb_iterator *it, const int expected_i);
 static int
 custom_key_comparator(const data_config *cfg, slice key1, slice key2);
 
+// Extern prototype defined in the library.
+bool
+validate_key_in_range(const splinterdb *kvs, slice key);
```

Comment:
Create `src/splinterdb_private.h` and declare this there, as described in my earlier comment.

Reply:
@rtjohnso - Not sure this is the preferred way. See my previous response. I feel it's an over-kill refactoring in this case.